lib/datadog/core/environment/cgroup.rb

```diff
@@ -0,0 +1,46 @@
+require 'datadog/core/ext/environment'
```

Comment:
Git sadly did not recognize these as moved 😢, sorry about the harder-than-necessary review.

Reply:
It's because the added module depth changed the indentation, and thus every line is different.